x-pack/plugins/cases/public/components/custom_fields/text/configure.tsx

```diff
@@ -6,27 +6,41 @@
  */
 
 import React from 'react';
-import { UseField } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
-import { CheckBoxField } from '@kbn/es-ui-shared-plugin/static/forms/components';
+import { UseField, useFormData } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+import { CheckBoxField, TextField } from '@kbn/es-ui-shared-plugin/static/forms/components';
 import type { CaseCustomFieldText } from '../../../../common/types/domain';
 import type { CustomFieldType } from '../types';
 import * as i18n from '../translations';
 
 const ConfigureComponent: CustomFieldType<CaseCustomFieldText>['Configure'] = () => {
+  const [{ required }] = useFormData();
```

Comment:
I would say to type (`<{ required: boolean }>`) it to be future-proof. Wdyt?

Reply:
done here and in `toggle/configure.tsx` 👍